_pages/home/updating-to-version-2211.35/migration.md

```diff
@@ -1,116 +1,195 @@
+   
+   @import '@spartacus/styles/index';
+   ```
 
-Now you've migrated your app to Angular 19 and Spartacus 2211.35, but it's not the end of the migration process.
+1. Add individual imports.
 
-Apps migrated to Angular 19 are not configured exactly the same as the new Angular 19 apps. 
-It is good to modernize your app to look like a new Angular 19 app, because it will help with migrations to future next versions of Angular and Spartacus. So now please follow the last guide: [Modernize Spartacus app migrated from 2211.32 to 2211.35](./modernize-apps-migrated-from-2211.32-to-2211.35.md)
+   If your application directly imports specific Bootstrap classes in any of your stylesheets, replace those imports with the corresponding `spartacus` imports. For example, if you have an import such as the following:
 
+   ```scss
+   @import '~bootstrap/scss/reboot';
+   ```
 
+   You should replace it with the following:
 
+   ```scss
+   @import '@spartacus/styles/vendor/bootstrap/scss/reboot';
+   ```
 
+## Silencing Sass Deprecation Warnings
+
+Silencing the deprecation warnings for the Sass `@import` is necessary because `@import` is used in the Spartacus styles and in the Bootstrap 4 styles, which are imported by the Spartacus styles.
+
+If this action is not taken, version 19 Angular CLI pollutes the terminal with excessive deprecation warnings when you run `ng serve`, which makes the developer experience less pleasant.
+
+For more information, see the following:
+
+- [`@import` is Deprecated](https://sass-lang.com/blog/import-is-deprecated) in the official Sass documentation.
+- [Style preprocessor options](https://angular.dev/reference/configs/workspace-config#style-preprocessor-options) in the official Angular documentation.
+
+To silence Sass deprecation warnings, in the `architect > build > options > stylePreprocessorOptions` section of `angular.json`, add a property with the object `"sass": { "silenceDeprecations":  ["import"] }`, as shown in the following example:
+
+```ts
+              "stylePreprocessorOptions": {
+                "includePaths": ["node_modules/"],
+                "sass": {
+                  "silenceDeprecations": ["import"]
+                }
+              }
+```
+
+## Modernizing Your Migrated Angular 19 Storefront App
+
+Storefront apps that are migrated to Angular 19 are not configured in exactly the same way as newly created Angular 19 apps. It is highly recommended that you modernize your app to be as similar as possible to a new Angular 19 app. This will help with updating to new versions of Angular and Spartacus in the future.
+
+Spartacus includes specially-prepared schematics to automatically modernize your app to be as similar as possible to a newly created Angular 19 app. To modernize your app using these schematics, run the following command from your project root directory:
+
+```bash
+ng g @spartacus/schematics:modernize-app-migrated-from-2211_32-to-2211_35
```

Comment:
```suggestion
ng g @spartacus/schematics:modernize-app-migrated-from-2211_32-to-2211_36
```

Reply:
I'll just point out, in the end I decided to add a note at the beginning of the procedure to say that all of the commands should be run from the root directory of your project, rather than including it at the end of every step.
Thank you for clarifying this, @Platonn 🙂